test/e2e/cert_manager_deployment_test.go

```diff
@@ -39,16 +46,224 @@ func TestSelfSignedCerts(t *testing.T) {
 
 	err := wait.PollImmediate(PollInterval, TestTimeout, func() (bool, error) {
 		// TODO: The loader.KubeClient might be worth splitting out. Let's see once we have more tests.
-		secret, err := loader.KubeClient.CoreV1().Secrets("sandbox").Get(context.TODO(), "root-secret", metav1.GetOptions{})
+		secret, err := loader.KubeClient.CoreV1().Secrets("hello").Get(ctx, "root-secret", metav1.GetOptions{})
 		if errors.IsNotFound(err) {
 			t.Logf("Unable to retrieve the root secret: %v", err)
 			return false, nil
 		}
 		if err != nil {
 			return false, err
 		}
+		return library.VerifySecretNotNull(secret)
+	})
+	require.NoError(t, err)
+}
+
+func TestACMECertsIngress(t *testing.T) {
+	ctx := context.Background()
+	loader := library.NewDynamicResourceLoader(ctx, t)
+	config, err := library.GetConfigForTest(t)
+	require.NoError(t, err)
+
+	loader.CreateFromFile(testassets.ReadFile, filepath.Join("testdata", "acme", "clusterissuer.yaml"))
+	defer loader.DeleteFromFile(testassets.ReadFile, filepath.Join("testdata", "acme", "clusterissuer.yaml"))
+	loader.CreateFromFile(testassets.ReadFile, filepath.Join("testdata", "acme", "hello.yaml"))
```

Comment:
please call this `namespace.yaml`

Reply:
and ensure it is properly initialized before continuing, xref'ing the e2e test framework: https://github.com/kubernetes/kubernetes/blob/f578b9a40d26973cf4122ec5b04fa9164adea158/test/e2e/framework/framework.go#L251

Reply:
Created a util function like the framework